### ISSUE TITLE
Missing autoIncrement: true on Postgres

### ISSUE THREAD

**Issue by danielfalsetti** [sequelize/sequelize-auto#370]
Issue:
autoIncrement: true is missing on generated models.

Env: Windows 10
Version: "sequelize": "^5.8.12",
    "sequelize-auto": "^0.4.29",

Table: 
CREATE TABLE public."TB_BANK" (
	"ID" int8 NOT NULL GENERATED ALWAYS AS IDENTITY,
	"S_NAME" varchar(100) NOT NULL,
	"N_PUBLIC_NUMBER" int4 NOT NULL,
	"updatedAt" timestamp NULL,
	"createdAt" timestamp NULL,
	CONSTRAINT "TB_BANK_pkey" PRIMARY KEY ("ID")
);

Generated class

/* jshint indent: 2 */

module.exports = function(sequelize, DataTypes) {
  return sequelize.define('TB_BANK', {
    ID: {
      type: DataTypes.BIGINT,
      allowNull: false,
      primaryKey: true
    },
    S_NAME: {
      type: DataTypes.STRING,
      allowNull: false
    },
    N_PUBLIC_NUMBER: {
      type: DataTypes.INTEGER,
      allowNull: false
    },
    updatedAt: {
      type: DataTypes.DATE,
      allowNull: true
    },
    createdAt: {
      type: DataTypes.DATE,
      allowNull: true
    }
  }, {
    tableName: 'TB_BANK'
  });
};

This is an issue?

**Comment by FredLackeyOfficial** [comment]
@danielfalsetti : Take a look at [`file-line-replacer`](https://github.com/FredLackey/file-line-replacer) when you have a moment.  I've had the same problem for years and cannot stand dealing with writing scripts to tweak the final files.  I created [`file-line-replacer`](https://github.com/FredLackey/file-line-replacer) to make fixing these files easier.  This allows sequelize-auto to be used, without issue, and then adjust the files when you're finished.

Here are the steps I use.  First, install it (I prefer installing it globally)...

`npm i -g file-line-replacer`

... and then, from inside of the project using sequalize, I run ... 

```
file-line-replacer \
  --search-dir "/Users/flackey/my-project/src/data/models" \
  --backup-dir "/Users/flackey/my-project/_backup" \
  --old-lines "allowNull: false,|primaryKey: true" \
  --new-lines "autoIncrement: true,|primaryKey: true" \
  --overwrite
``` 
The above command will tweak your models, adding the `autoIncrement` key, and create a backup for each file before it is modified.

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`  
Please try it and close this issue if so.  Thanks!
